Lib/ufo2ft/featureWriters/gdefFeatureWriter.py

```diff
@@ -14,18 +16,25 @@ class GdefFeatureWriter(BaseFeatureWriter):
     tableTag = "GDEF"
 
     def setContext(self, font, feaFile, compiler=None):
-        ctx = super().setContext(font, feaFile, compiler=compiler)
+        ctx = self.context = SimpleNamespace(
```

Comment:
why not call super().setContext any more here? is it to avoid the insert markers collection? Maybe the latter should be moved out of the base class setContext if it's not relevant to all sub-classes

Reply:
I modified baseFeatureWriter to skip insert marker collection when insertFeatureMarker is None.